pkg/standalone/runfileconfig/run_file_config_parser_test.go

```diff
@@ -133,3 +133,47 @@ func TestGetBasePathFromAbsPath(t *testing.T) {
 		})
 	}
 }
+
+func TestResolveAppCMD(t *testing.T) {
+	tempBaseDir, err := os.MkdirTemp("", "test")
```

Comment:
Can we mock the filesystem here as well? Re: our conversation in #1158

Reply:
@shubham1172 I will do test refactoring as part of #1158. It will be easier to do it one PR as that already has some of these test refactoring. I tried for one of the tests here but it seems other tests also I need to modify.

Reply:
i agree that the change for mocking filesystem should be done separately.